app/models/Ticket.scala

```diff
@@ -0,0 +1,21 @@
+package models
+
+case class Ticket(rowNumber: Int, seatNumberInRow: Int, ticketType: TicketType) {
+  override def toString: String = s"Row number: $rowNumber <--> seat number: ${seatNumberInRow + 1} <--> ticket type: $ticketType <--> ticket's price: ${ticketType.price} zł"
+}
+
+abstract class TicketType {
+  val price: Double
```

Comment:
why do you think Double is a good choice for price representation?

Reply:
In real life program I think it would be better to use BigDecimal - it has more precise methods for rounding (it would be important when dealing with VAT taxes or things like that). I chose Double for this demo, because in this particular case there is no hard calculation that would need the usage of special classes like BigDecimal. Just three prices - 12.50 , 18 and 25.